usr/bin/regolith-session-sway

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# This script initializes Regolith Xresources and launches i3.
+
+set -Eeu -o pipefail
+
+source /usr/lib/regolith/regolith-session-common.sh
+
+# --------- Script Main -----------
+resolve_default_config_file
+
+# Preloads user environment variables
+USER_ENVIRONMENT=${trawlcat sway.preload_env $HOME/.config/regolith2/sway/environment}
```

Comment:
I would prefer if this could be expressed in a standard way if possible.

https://wiki.gnome.org/Initiatives/Wayland/SessionStart

https://github.com/sddm/sddm/issues/916

https://superuser.com/questions/992210/how-to-set-environment-variables-for-a-gnome-wayland-session

Hmm, seems like maybe Wayland doesn't provide anything (yet)..?  Are you aware of anything that my 5 mins of google searching didn't uncover?  If not let's keep this logic but remove the default that has the moz and jdk stuff.